runtime/render_a/src/sys/d3d/d3d_renderstyleinterface.cpp

```diff
@@ -62,7 +62,7 @@ CRenderStyle* D3DRenderStyles::LoadRenderStyle(const char* szFilename)
 					pFileStream->Release(); 
 					string szTmp = "Couldn't Load Renderstyle "; 
 					szTmp += szFilename; szTmp += "\n"; 
-					OutputDebugString(szTmp.c_str()); 
+					OutputDebugString("%s", szTmp.c_str()); 
```

Comment:
This is causing an error in Visual Studio, and OutputDebugString does not take 2 arguments. This change should be reverted.

Reply:
made the macro on linux similar to the Windows function